### PR TITLE
[reporting] introduce PDF reports | DBNBK-119, CRP-841

### PR DESCRIPTION
**Pull request by jcmaunsell**
Reverts DataDog/documentation#22868, which reverted https://github.com/DataDog/documentation/pull/21736. Replaces production images with demo org images.

### CODE REVIEW

File: content/en/dashboards/scheduled_reports.md [hunk 1]
```diff
@@ -18,77 +18,88 @@ aliases:
...
+{{< img src="dashboards/scheduled_reports/scheduled_reports_configuration_modal.png" alt="The configuration modal for scheduled reports, with two reports displayed, each showing their titles, tags, recipients, frequency, an option to toggle the report on or off, and a button to edit the report. At the bottom is a button to add a new report and a done button" style="width:90%;" >}}
 
 ## Permissions
 
-Only users with the **Dashboard Report Write** permission can generate a report. This permission is turned on by default for Admins and off for all other roles.
+Users need the `generate_dashboard_reports` [permission][29] to create and edit report schedules. This permission can be granted by another user with the `user_access_manage` permission and is available by default to users with the `org_management` permission.
```
Comment (jcmaunsell) [inline content/en/dashboards/scheduled_reports.md:65]: refers to permissions by `snake_case` name to match the [permissions docs](https://docs-staging.datadoghq.com/revert-22868-revert-21736-esther/docs-6980-scheduled-reports-update/account_management/rbac/permissions/#dashboards)

File: content/en/dashboards/scheduled_reports.md [hunk 2]
```diff
@@ -18,77 +18,88 @@ aliases:
...
 
 {{< site-region region="gov" >}}
-<div class="alert alert-warning">Scheduled Reports are not supported for this site.</div>
+<div class="alert alert-warning">Scheduled Reports are not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
 {{< /site-region >}}
 
 ## Overview
 
-Scheduled dashboard reports automatically send a visual summary of a dashboard to selected recipients on a schedule. The generated reports display images of widgets from a dashboard in a linear HTML format.
+Scheduled reports enable Datadog users to share dashboards as high-density PDFs by email on a recurring basis.
```
Comment (jcmaunsell) [inline content/en/dashboards/scheduled_reports.md:26]: **[major feature change]** reports are now PDF attachments, not embedded HTML!

File: content/en/dashboards/scheduled_reports.md [hunk 3]
```diff
@@ -18,97 +18,83 @@ aliases:
...
 
-{{< img src="dashboards/scheduled_reports/report_management.png" alt="The Report Management setting under the Settings tab in Public Sharing within Organization Settings in Datadog with the setting Enabled" style="width:100%;" >}}
+The following widget types are **not** supported:
+- [Embedded App][embedded-app-widget]
+- [Experimental][experimental-widget]
+- [Iframe][iframe-widget]
+- [Image][image-widget]
+- [Hostmap][hostmap-widget]
+- [Run Workflow][run-workflow-widget]
+- [Wildcard][wildcard-widget]
```
Comment (jcmaunsell) [inline content/en/dashboards/scheduled_reports.md:85]: https://github.com/DataDog/dogweb/blob/52dc55daba8b071d82183df0ef2f61834f49e746/dogweb/model/dashboard_widget/lib/widget_manager.py#L207-L217

https://datadoghq.atlassian.net/wiki/spaces/DB/pages/3349708842/PDF+Reports+Supported+Widgets+Data+Sources

File: content/en/dashboards/scheduled_reports.md [hunk 4]
```diff
@@ -18,97 +18,83 @@ aliases:
...
-[17]: /account_management/users
+[embedded-app-widget]: /dashboards/widgets/embedded_app/
+[experimental-widget]: /dashboards/widgets/experimental/
+[iframe-widget]: /dashboards/widgets/iframe/
+[image-widget]: /dashboards/widgets/image/
+[hostmap-widget]: /dashboards/widgets/hostmap/
+[permissions]: /account_management/rbac/permissions/
+[public-sharing]: /account_management/org_settings/#public-sharing
+[run-workflow-widget]: /dashboards/widgets/run_workflow/
+[wildcard-widget]: /dashboards/widgets/wildcard/
```
Comment (jcmaunsell) [inline content/en/dashboards/scheduled_reports.md:99]: it's easier to keep track of markdown links using names than with numbers

File: content/en/dashboards/scheduled_reports.md [hunk 5]
```diff
@@ -18,97 +18,83 @@ aliases:
...
-[10]: /dashboards/widgets/slo/
-[11]: /dashboards/widgets/slo_list/
-[12]: /dashboards/widgets/table/
-[13]: /dashboards/widgets/timeseries/
-[14]: /dashboards/widgets/top_list/
-[15]: /account_management/org_settings/#public-sharing
-[16]: /dashboards/widgets/pie_chart/
-[17]: /account_management/users
+[embedded-app-widget]: /dashboards/widgets/embedded_app/
+[experimental-widget]: /dashboards/widgets/experimental/
```
Comment (jcmaunsell) [inline content/en/dashboards/scheduled_reports.md:92]: remove this (internal only)

File: content/en/dashboards/scheduled_reports.md [hunk 6]
```diff
@@ -18,97 +18,83 @@ aliases:
...
 
-{{< img src="dashboards/scheduled_reports/scheduled_reports_configuration_modal.png" alt="The configuration modal for scheduled reports, with two reports displayed, each showing their titles, tags, recipients, frequency, an option to toggle the report on or off, and a button to edit the report. At the bottom is a button to add a new report and a done button" style="width:100%;" >}}
+{{< img src="dashboards/scheduled_reports/scheduled_reports_configuration_modal.png" alt="The configuration modal for scheduled reports, with two reports displayed, each showing their titles, tags, recipients, frequency, an option to toggle the report on or off, and a button to edit the report. At the bottom is a button to add a new report and a done button" style="width:90%;" >}}
 
 ## Permissions
 
-Only users with the **Dashboard Report Write** permission can generate a report. This permission is turned on by default for Admins and off for all other roles.
+Users need the `generate_dashboard_reports` [permission][permissions] to create and edit report schedules.
```
Comment (jcmaunsell) [inline content/en/dashboards/scheduled_reports.md:67]: back to **Dashboards Report Write**

File: content/en/dashboards/scheduled_reports.md [hunk 7]
```diff
@@ -18,97 +18,83 @@ aliases:
...
+This permission can be granted by another user with the `user_access_manage` permission and is available by default to users with the `org_management` permission.
+
+{{< img src="dashboards/scheduled_reports/dashboard_permissions.png" alt="A screenshot of an individual user's permissions from within the organization settings page. The dashboards report write permission is highlighted under the dashboards section" style="width:90%;" >}}
 
-Images generated in reports show all data regardless of granular read restrictions. Datadog recommends limiting the report permissions to users who have no granular read restrictions on data. To grant a user the **Dashboard Report Write** permission, create a new role with the **Dashboards Report Write** permission turned on and assign the user to this role. Alternatively, assign the **Admin** role to this user. To learn about managing roles and permissions, see [User Management][17]. 
+Users with the `org_management` permission can enable or disable the scheduled reports feature for their organization from the **Settings** tab under [Public Sharing][public-sharing] in **Organization Settings**.
```
Comment (jcmaunsell) [inline content/en/dashboards/scheduled_reports.md:72]: fix `snake_case`

File: content/en/dashboards/scheduled_reports.md (hunk 3 above)
Comment (estherk15) [reply content/en/dashboards/scheduled_reports.md:85]: Can you also add a sentence on what would happen if your dashboard includes one of these unsupported widgets?

File: content/en/dashboards/scheduled_reports.md [hunk 8]
```diff
@@ -18,97 +18,81 @@ aliases:
...
+{{< img src="dashboards/scheduled_reports/scheduled_reports_configuration_modal.png" alt="The configuration modal for scheduled reports, with two reports displayed, each showing their titles, tags, recipients, frequency, an option to toggle the report on or off, and a button to edit the report. At the bottom is a button to add a new report and a done button" style="width:90%;" >}}
 
 ## Permissions
 
-Only users with the **Dashboard Report Write** permission can generate a report. This permission is turned on by default for Admins and off for all other roles.
+Users need the **Dashboards Report Write** [permission][permissions] to create and edit report schedules.
+This permission can be granted by another user with the **User Access Manage** permission and is available by default to users with the **Datadog Admin** [out-of-the-box role][ootb-role].
```
Comment (estherk15) [inline content/en/dashboards/scheduled_reports.md:70]: I understand why you changed the format of the reference links, but I can't justify making an exception for this page without considering the 5000+ pages we manage 🙏 

```suggestion
This permission can be granted by another user with the **User Access Manage** permission, and is available by default to users with the **Datadog Admin** [out-of-the-box role][ootb-role].
```

**Review by estherk15 — CHANGES_REQUESTED** [review]
Thanks for updating this! Left some feedback

File: static/images/dashboards/scheduled_reports/report_configuration_modal.png
[the remark this answers is not in the document]
Comment (jcmaunsell) [reply static/images/dashboards/scheduled_reports/report_configuration_modal.png:1]: yes they are! I edited the page source directly

File: content/en/dashboards/scheduled_reports.md [hunk 9]
```diff
@@ -18,97 +18,81 @@ aliases:
...
-- [SLO][10]
-- [SLO List][11]
-- [Table][12]
-- [Timeseries][13]
-- [Top List][14]
+Create a report from any [dashboard or timeboard][dashboard] that has at least one [supported widget](#unsupported-widget-types).
+
+Click the **Share** button at the top of your dashboard and select **Schedule a Report**.
+
+{{< img src="dashboards/scheduled_reports/report_configuration_modal.png" alt="The configuration modal for an individual dashboard report, with sections to set a schedule, add recipients, and customize email. At the bottom of the modal are buttons to edit template variables, delete report, send preview, cancel, and save" style="width:90%;" >}}
```
[the remark this answers is not in the document]
Comment (jcmaunsell) [reply content/en/dashboards/scheduled_reports.md:40]: responded - they're fake email addresses :)

File: static/images/dashboards/scheduled_reports/report_pdf.png
Comment (jcmaunsell) [inline static/images/dashboards/scheduled_reports/report_pdf.png:1]: demo org dashboard: https://app.datadoghq.com/dashboard/zmy-z2e-xv4

File: content/en/dashboards/scheduled_reports.md (hunk 3 above)
Comment (jcmaunsell) [reply content/en/dashboards/scheduled_reports.md]: done!

**Review by estherk15 — APPROVED** [review]

**Pull request merged**